### DISCUSSION TITLE
animation timeScale property doesn't seem to work

### DISCUSSION THREAD

**Discussion by furkandindar** [google/model-viewer#3154] [Q&A] [answered]
Hello,

I am trying to implement reverse animation by using timeScale property set to -1. Following the link [Change animation speed](https://modelviewer.dev/examples/animation/index.html#controlSpeed) but even though it works on that example, it does not work on my [codesandbox](https://codesandbox.io/s/bold-bush-3zmm3?file=/index.html). Is there something i'm missing here? I directly copied and pasted the example code to the codesandbox. Thanks in advance!

**Comment by samaneh-kazemi** [comment, marked as answer]
Hi @furkandindar, thanks for reaching out. This feature was [merged after the v1.10](https://github.com/google/model-viewer/pulls?q=is%3Apr+is%3Aclosed) release and is not available on npm/unpkg yet. You're welcome to build the latest version of model-viewer locally and try any pre-release feature like this. Otherwise, wait for the next release. Hope this helps.

![Screen Shot 2022-01-26 at 10 37 31 AM](https://user-images.githubusercontent.com/3915613/151226223-b09a128d-7d04-41e7-b5ab-975292539868.png)
